week13/상현/PG_GPS.java

```diff
@@ -0,0 +1,67 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+/**
+ * 실패...
+ * dp + dfs로 시도
+ */
+class Solution {
+    private static List<Integer>[] trees;
+    private static int[][] dp;
+    public int solution(int n, int m, int[][] edge_list, int k, int[] gps_log) {
+        int answer = 0;
+
+        trees = new ArrayList[n + 1];
+        dp = new int[n + 1][k];
+        for (int i = 1; i <= n; i++) {
+            trees[i] = new ArrayList<>();
+            Arrays.fill(dp[i], Integer.MAX_VALUE - k);
```

Comment:
```suggestion
            Arrays.fill(dp[i], -1);
```
아래에 설명 있습니다!
이미 계산 된 것과 아예 방문되지도 않았다는 것을 표시하기 위해 -1로 초기화했습니다